tsconfig.json

```diff
@@ -49,6 +51,7 @@
     /* Experimental Options */
     // "experimentalDecorators": true,        /* Enables experimental support for ES7 decorators. */
     // "emitDecoratorMetadata": true,         /* Enables experimental support for emitting type metadata for decorators. */
+    "allowUnreachableCode": true
```

Comment:
why did you add this?

Reply:
In some moment webpack was generating some unreachable code like `if (true) {} else {}` and I set this option to avoid the error on the ts compilation but now seems that we can get rid of it.